js/apps/system/_admin/aardvark/APP/frontend/js/templates/shardsView.ejs

```diff
@@ -86,7 +86,7 @@
                     <% if (shard.progress.total === 0) { %>
                       <% percent = numeral(0).format('0.00%'); %>
                     <% } else { %>
-                      <% percent = numeral(shard.progress.current / shard.progress.total).format('0.00%'); %>
+                      <% percent = numeral(shard.progress.current / shard.progress.total).format('0.00%'); if (percent === "100.00%") { percent = "99.99%"; } %>
```

Comment:
looks like yet another workaround... and why display "99.99%" when the proper result would be 100% (e.g. for 1 / 1)?
Can't we fix this properly and by simply *truncating* the value and not *rounding * it?
For example:
```js
percent = (shard.progress.current / shard.progress.total * 100).toString().match(/^-?\d+(?:\.\d{0,2})?/)[0] + '%';
```